automated install/basic-install.sh

```diff
@@ -1876,7 +1876,7 @@ installLogrotate() {
     local str="Installing latest logrotate script"
     printf "\\n  %b %s..." "${INFO}" "${str}"
     # Copy the file over from the local repo
-    install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/advanced/Templates/logrotate /etc/pihole/logrotate
+    install -D -m 644 -T "${PI_HOLE_LOCAL_REPO}"/advanced/Templates/logrotate "${PI_HOLE_CONFIG_DIR}"/pihole/logrotate
```

Comment:
`"${PI_HOLE_CONFIG_DIR}"/pihole/logrotate` needs to be `"${PI_HOLE_CONFIG_DIR}"/logrotate`, but I guess a rebase would be required in the first place if this PR shall be considered 😉.